runner/operator/access/v2_1_0/snps_and_indels/input_template.json.jinja2

```diff
@@ -0,0 +1,160 @@
+      "tn_ratio_thres": 2,
+      "tier_one_alt_min": 3,
+      "tier_two_alt_min": 5,
+      "tumor_detect_alt_thres": 2,
+      "curated_detect_alt_thres": 2,
+      "DS_tumor_detect_alt_thres": 2,
+      "DS_curated_detect_alt_thres": 2,
+      "tumor_vaf_germline_thres": 0,
+      "normal_vaf_germline_thres": 0.3,
+      "min_n_curated_samples_alt_detected": 2
+    },
+    "dbsnp": {
+        "class": "File",
+        "location": "juno:///juno/work/access/production/resources/dbSNP/versions/b37/dbsnp_137.b37.vcf"
+    },
+    "cosmic": {
+        "class": "File",
+        "location": "juno:///juno/work/access/production/resources/cosmic/versions/v67/CosmicCodingMuts_v67_b37_20131024__NDS.vcf"
+    },
+    "ref_fasta": {
+        "class": "File",
+        "location": "juno:///juno/work/access/production/resources/reference/versions/hg19_virus_special/hg19_virus.fasta"
+    },
+    "exac_filter": {
+        "class": "File",
+        "location": "juno:///juno/work/access/production/resources/exac/versions/0.3.1/ExAC_nonTCGA.r0.3.1.sites.vep.vcf.gz"
+    },
+    "hotspots": {
+        "class": "File",
+        "location": "juno:///juno/work/access/production/resources/msk-access/v1.0/regions_of_interest/versions/v1.0/hotspot-list-union-v1-v2_with_TERT.txt"
+    },
+    "blacklist_file": {
+        "class": "File",
+        "location": "juno:///juno/work/access/production/resources/msk-access/v1.0/regions_of_interest/versions/v1.0/ACCESS_blocklist_26_10_2022.txt"
+    },
+    "custom_enst_file": {
+        "class": "File",
+        "location": "juno:///juno/work/access/production/resources/msk-access/v1.0/regions_of_interest/versions/v1.0/dmp_ACCESS-panelA-v1-isoform-overrides"
+    },
+    "bed_file": {
+        "class": "File",
+        "location": "juno:///juno/work/access/production/resources/msk-access/v2.0/regions_of_interest/versions/v1.0/MSK-ACCESS-v2_canonicaltargets_500buffer.bed"
+    },
+    "annotate_concat_header_file": {
+        "class": "File",
+        "location": "juno:///juno/work/access/production/resources/msk-access/v1.0/regions_of_interest/versions/v1.0/mutect_annotate_concat_header.txt"
+    },
+    "curated_duplex_bams": {
+        "novaseq": {
+            "class": "Directory",
+            "location": "juno:///juno/work/access/production/resources/msk-access/v1.0/novaseq_curated_duplex_bams_dmp"
+        },
+        "hiseq": {
+            "class": "Directory",
+            "location": "juno:///juno/work/access/production/resources/msk-access/v1.0/hiseq4000_curated_duplex_bams_dmp"
+        }
+    },
+    "curated_simplex_bams": {
+        "novaseq": {
+            "class": "Directory",
+            "location": "juno:///juno/work/access/production/resources/msk-access/v1.0/novaseq_curated_simplex_bams_dmp"
+        },
+        "hiseq": {
+            "class": "Directory",
+            "location": "juno:///juno/work/access/production/resources/msk-access/v1.0/hiseq4000_curated_simplex_bams_dmp"
+        }
+    },
```

Comment:
Maybe these are place holders but, these need to be updated to v2.0 folders in `/juno/cmo/access/production/resources/msk-access/v2.0`, also we don't have hiseq anymore